src/lesson5/HomeWork5.java

```diff
@@ -0,0 +1,27 @@
+package lesson5;
+
+
+public class HomeWork5 {
+
+protected static final int PERS = 5;
+
+    public static void main(String[] args) {
+
+        Worker[] persArray = new Worker[PERS];
+
+        for (int i = 0; i <= PERS-1; i++){
```

Comment:
так чуть проще и стандартней
```suggestion
        for (int i = 0; i < PERS; i++){
```